addon/tern/tern.js

```
@@ -0,0 +1,531 @@
+      if (doc.doc.somethingSelected())
+        query.start = doc.doc.getCursor("start");
+    }
+    var startPos = query.start || query.end;
+
+    if (doc.changed) {
+      if (doc.doc.lineCount() > bigDoc && allowFragments !== false &&
+          doc.changed.to - doc.changed.from < 100 &&
+          doc.changed.from <= startPos.line && doc.changed.to > query.end.line) {
+        files.push(getFragmentAround(doc, startPos, query.end));
+        query.file = "#0";
+        var offsetLines = files[0].offsetLines;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'offsetLines'. Consider to delete it or rename it.

'offsetLines' is defined here:


(lint/suspicious/noRedeclare)